contracts/adapters/GammaAdapter.sol

```diff
@@ -661,19 +661,10 @@ contract GammaAdapter is IProtocolAdapter, DSMath {
             underlying = WETH;
         }
 
-        // Put otokens have USDC as the backing collateral
-        // so we can ignore the collateral asset passed in option terms
-        address collateralAsset;
-        if (isPut) {
-            collateralAsset = USDC;
-        } else {
-            collateralAsset = underlying;
-        }
```

Comment:
Remove these because the assumption that puts can only be collateralized either by USDC or the `underlying` is incorrect. Otokens can be collateralized with yTokens for example.

Reply:
👍